web/realtime/realtime.go

```diff
@@ -283,7 +286,34 @@ func ws(c echo.Context) error {
 	}
 }
 
+// Notify is the API handler for POST /realtime/:doctype/:id: this route can be
+// used to send documents in the real-time without having to persist them in
+// CouchDB.
+func Notify(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+	doctype := c.Param("doctype")
+	id := c.Param("id")
+
+	if err := permission.CheckReadable(doctype); err != nil {
```

Comment:
Should we check that the doctype is Writable ?

Reply:
I don't think so. For example, the `io.cozy.jobs` are not writable, but I think it is OK to send real-time docs about them.